app/src/test/java/com/github/se/eventradar/ChatViewModelUnitTest.kt

```diff
@@ -299,6 +299,25 @@ class ChatViewModelUnitTest {
     assert(viewModel.uiState.value.messageHistory == (expected as Resource.Success).data)
   }
 
+  @Test
+  fun `observeMessages Success No Existing Message History`() = runTest {
```

Comment:
Suggestion: Consider deleting this test which seems to be extremely similar to : `observeMessages Success`